types/cytoscape-dagre/index.d.ts

```diff
@@ -0,0 +1,64 @@
+// Type definitions for cytoscape-dagre 2.3
+// Project: https://github.com/cytoscape/cytoscape.js-dagre
+// Definitions by: Felix Barczewicz <https://github.com/DieserFelix>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
```

Comment:
```suggestion
import cytoscape = require('cytoscape');

```
I believe you need this, it just works now, as there is direct import in test file

Reply:
Oh, you are absolutely correct. Added it in the latest commit. Thank you!